integrations/fastembed/src/haystack_integrations/components/rankers/fastembed/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .ranker import FastembedRanker
```

Comment:
the existing ranker should not be removed from here.

also, please not introduce new init files: integrations/fastembed/src/haystack_integrations/__init__.py, etc.
This package is a [namespace package](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/), so these init files should be avoided.